library/src/com/google/maps/android/geojson/GeoJsonFeature.java

```diff
@@ -0,0 +1,211 @@
+package com.google.maps.android.geojson;
+
+import com.google.android.gms.maps.model.LatLngBounds;
+
+import java.util.HashMap;
+import java.util.Observable;
+import java.util.Observer;
+
+/**
+ * A GeoJsonFeature has a geometry, bounding box, id and set of properties. Styles are also stored
+ * in this class.
+ */
+public class GeoJsonFeature extends Observable implements Observer {
+
+    private final String mId;
+
+    private final LatLngBounds mBoundingBox;
+
+    private final HashMap<String, String> mProperties;
+
+    private GeoJsonGeometry mGeoJsonGeometry;
+
+    private GeoJsonPointStyle mGeoJsonPointStyle;
+
+    private GeoJsonLineStringStyle mGeoJsonLineStringStyle;
+
+    private GeoJsonPolygonStyle mGeoJsonPolygonStyle;
+
+    /**
+     * Creates a new GeoJsonFeature object
+     *
+     * @param GeoJsonGeometry type of geometry to assign to the feature
+     * @param id              id to refer to the feature by
+     * @param properties      hashmap of data containing properties related to the feature
+     * @param boundingBox     array defining the bounding box of the feature
```

Comment:
Any reason why the bounding box is user-provided, rather than being calculated from the geometry?